modules/serverless-broker-types.adoc

```diff
@@ -1,27 +1,24 @@
 // Module included in the following assemblies:
 //
-// * /serverless/develop/serverless-using-brokers.adoc
+// * /serverless/discover/serverless-brokers.adoc
 
 :_content-type: CONCEPT
 [id="serverless-broker-types_{context}"]
 = Broker types
 
-There are multiple broker implementations available for use with {ServerlessProductName}, each of which have different event delivery guarantees and use different underlying technologies. You can choose the broker implementation when creating a broker by specifying a broker class, otherwise the default broker class is used. The default broker class can be configured by cluster administrators.
-// TO DO: Need to add docs about setting default broker class.
+Cluster administrators can set the default broker implementation for a cluster. When you create a broker, the default broker implementation is used, unless you provide set configurations in the `Broker` object.
 
-[id="serverless-broker-types-channel-based"]
-== Channel-based broker
-
-The channel-based broker implementation internally uses channels for event delivery. Channel-based brokers provide different event delivery guarantees based on the channel implementation a broker instance uses, for example:
+:FeatureName: Kafka broker
+include::snippets/technology-preview.adoc[leveloffset=+2]
 
-* A broker using the `InMemoryChannel` implementation is useful for development and testing purposes, but does not provide adequate event delivery guarantees for production environments.
+[id="serverless-broker-types-default_{context}"]
+== Default broker implementation for development purposes
 
-* A broker using the `KafkaChannel` implementation provides the event delivery guarantees required for a production environment.
+Knative provides a default, channel-based broker implementation. This channel-based broker can be used for development and testing purposes, but does not provide adequate event delivery guarantees for production environments. The default broker is backed by the `InMemoryChannel` channel implementation by default.
 
-[id="serverless-broker-types-kafka"]
-== Kafka broker
+It is not recommended to configure the channel-based broker to be backed by the `KafkaChannel` channel implementation. If you want to use Kafka, it is recommended to use the Kafka broker implementation instead to reduce network hops.
```

Comment:
We're not supposed to use "[recommended](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#r)."

Also, I flipped the sentence to put the positive one first. 

Of course, not knowing anything about this content, I may have totally messed up your intention. I hope not.

"If you want to use Kafka to reduce network hops, use the Kafka broker implementation. Do not configure the channel-based broker to be backed by the `KafkaChannel` channel implementation."

Reply:
Good suggestion, thanks!